package.json

```diff
@@ -38,21 +38,21 @@
   "dependencies": {},
   "devDependencies": {
     "browser-perf": "1.2.3",
-    "chromedriver": "2.13.0",
+    "chromedriver": "2.19.0",
     "commitplease": "2.0.0",
     "ejs-template": "0.1.0",
     "grunt": "0.4.5",
     "grunt-autoprefixer": "2.1.0",
-    "grunt-contrib-cssmin": "0.10.0",
-    "grunt-contrib-csslint": "0.4.0",
-    "grunt-contrib-jshint": "0.10.0",
+    "grunt-contrib-cssmin": "0.14.0",
+    "grunt-contrib-csslint": "0.5.0",
+    "grunt-contrib-jshint": "0.11.3",
     "grunt-contrib-connect": "0.9.0",
     "grunt-contrib-watch": "0.6.1",
     "grunt-csscomb": "3.0.0",
     "grunt-git-authors": "2.0.0",
     "grunt-html": "1.6.0",
     "grunt-jscs": "2.0.0",
-    "grunt-sass": "0.17.0",
+    "grunt-sass": "1.1.0-beta",
```

Comment:
Is there no stable version available that works on node 4? Or what's the reason for including a beta version?